apps/extension/src/background/ports/completion.ts

```diff
@@ -56,6 +59,12 @@ const handler: PlasmoMessaging.PortHandler<
       ? [getOutput(txn.input, replies.join(""))]
       : undefined
     txn.error = errors.join("") || undefined
+
+    const totalCost = transactionManager.getTotalCost(txn, caller.config.getPromptCost(), caller.config.getCompletionCost())
```

Comment:
@alexanderatallah I need the `request: RequestData` variable here so that I can call getPromptCost and getCompletionCost based on the model being used. How can I access that variable in this file?